src/v3/src/types/schema.ts

```diff
@@ -142,9 +142,13 @@ export function isUISchemaLayoutType(type: string): boolean {
 export interface FieldElement extends UISchemaElement {
   type: 'Field';
   key: string;
+  /**
+   * @description TODO: temporary solution for custom fields in profile enrollment
+   */
+  required?: boolean | undefined;
```

Comment:
Which part is the temporary bit? And what is needed to remediate this?

Reply:
Ah, so I added this `required` prop as a temp solution to denote the required vs optional props in the profile enrollment flow only. 

So just this `required` field (in this file) is the temp solution. In the transformer the function to update this prop is also apart of the temporary solution. 

I spoke with ODS on this, and the long term solution is after they wrap MUI components, they will provide a optional attribute that will display a label above the input (on the right side) similar to v2. 

Once we implement their wrapped components we can get rid of this and use the required prop directly apart of `inputMeta`